### ISSUE TITLE
adapt-contrib-trickle: Error when document.activeElement = null;

### ISSUE THREAD

**Issue by oliverfoster** [adaptlearning/adapt_framework#2520]
### Subject of the issue
New code introduces bug as cannot lookup function on null document.activeElement.
